contributing/community/reviews.rst

```diff
@@ -0,0 +1,190 @@
+Community Reviews
+=================
+
+Symfony is an open-source project driven by a large community. If you don't feel
+ready to contribute code or patches, reviewing issues and pull requests (PRs)
+can be a great start to get involved and give back.
+
+Why Reviewing is Important
+--------------------------
+
+Community reviews are essential for the development of the Symfony framework,
+since there are many more pull requests and bug reports than there are members
+in the Symfony core team to review, fix and merge them.
+
+On the `Symfony issue tracker`_, you can find many items in a "Needs Review"
+status:
+
+* **Bug Reports**: Bug reports primarily need to be checked for completeness.
+  Is any important information missing? Can the bug easily be reproduced?
+
+* **Pull Requests**: Pull requests contain code that fixes a bug or implements
+  new functionality. Reviews of pull requests ensure that they are implemented
+  properly, are covered by test cases, don't introduce new bugs and maintain
+  backwards compatibility.
+
+Note that **anyone who has some basic familiarity with Symfony and PHP can
+review bug reports and pull requests**. You don't need to be an expert to help.
+
+Be Constructive
+---------------
+
+Before you begin, remember that you are looking at the result of someone else's
+hard work. A good review post thanks the contributor for their work, identifies
+what was done well, identifies what should be improved, and suggests a next
+step.
+
+Create a GitHub Account
+-----------------------
+
+Symfony uses GitHub_ to manage bug reports and pull requests. If you want to
+do reviews, you need to `create a GitHub account`_ and log in.
+
+The Bug Report Review Process
+-----------------------------
+
+A good way to get started with reviewing is to pick a bug report from the
+`bug reports in need of review`_.
+
+The steps for the review are:
+
+1. **Is the Report Complete?**
+
+   Good bug reports contain a link to a fork of the `Symfony Standard Edition`_
+   (the "reproduction project") that reproduces the bug. If it doesn't, the
+   report should at least contain enough information and code samples to
+   reproduce the bug.
+
+2. **Reproduce the Bug**
+
+   Download the reproduction project and test whether the bug can be reproduced
+   on your system. If the reporter did not provide a reproduction project,
+   create one by forking_ the `Symfony Standard Edition`_. Reproduce the bug
+   with the instructions given by the reporter.
+
+3. **Update the Issue Status**
+
+   At last, add a comment and update the status of the bug report. **Thank the
+   reporter for reporting the bug**. Include the line ``Status: <status>`` in
```

Comment:
Do we need to explain that this format is necessary as it will be used to automatically change the state of the issue? I'm not sure if everyone understands this from just reading these lines and they may be confused by not finding any button to change the state.